src/capacitor/pager.ts

```diff
@@ -78,6 +80,25 @@ export class Pager {
     }
   }
 
+  restrictTotalRecords(total: number, pageSize: number): number {
+    const remainder = total % pageSize;
```

Comment:
probably don't need to calculate the remainder until line 91 as it's  only used there. fine to keep it as a separate variable, but I feel like it should only be at the top if other things are going to use it